drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -87,13 +115,17 @@
             <section class="tab-content">
                 <form class="display-labels-form">
                     <h3>{% trans "Zone labels" %}</h3>
-                    <label for="display-labels">{% trans "Display label names on the image" %}:</label>
-                    <input name="display-labels" id="display-labels" type="checkbox" />
+                    <label>
+                        <span>{% trans "Display label names on the image" %}:</span>
```

Comment:
Include the colon in the text to be translated-- colons won't make sense in all languages. Or is it more natural to have the checkbox before the text, so there is no need for a colon?